quickstart1/app/Http/Controllers/QuickStartController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Task;
+use Illuminate\Http\Request;
+use Illuminate\Database;
+class QuickStartController extends Controller
+{
+    public function loadTask()
+    {
+        $tasks = Task::all();
+        return view('QuickStart_view',compact('tasks'));
```

Comment:
1. trước return có 1 dòng trắng 
2. Cần có 1 space trước và sau các toán tử như +, -, *, /, ., >, <, == ...